src/chat.rs

```diff
@@ -3566,7 +3566,7 @@ pub(crate) async fn add_contact_to_chat_ex(
         if chat.is_protected() && !contact.is_verified(context).await? {
```

Comment:
At least i see no strong reason to forbid adding unidir-verified contacts to protected chats in the core. If a UI wants, it can hide such contacts anyway. But i'd prefer to let a user decide and just explain in FAQ in which cases it may not work

Reply:
But it's actually just a restricting check -- we already have all info to add the member to protected chats. I think, "correctly implemented UIs" isn't the right wording, it's just a choice of a UI. As for the tests, they should check the addition result on the side of the added member.

But ok, currently we have no other UIs, so it doesn't really matter.